source/howto/CreateVMFromCommandLine.rst

```diff
@@ -4,11 +4,11 @@ Create a Virtual Machine from command line in STFC Openstack
 
 It’s common to want to create VMs from command line. To do this, you need to be on a host that has the openstack command line interface installed.
 
-On our Scientific Linux 7 machines the correct repository is already set up so you can use you install the client by running:
+On our Ubuntu Focal 20.04 image the correct repository is already available, so you can use you install the client by running:
```

Comment:
`On Ubuntu machines, we can simply install the client by running:`

(Since we're not going to have a Python 4 :crossed_fingers: )